### PR TITLE
Feat add style to offset selector

### PR DESCRIPTION
**Pull request by SejoB**
image
#362

### CODE REVIEW

File: src/components/atoms/AnywayMostSevereAccidentsMarker.tsx [hunk 1]
```diff
@@ -1,15 +1,52 @@
...
+import TooltipArrow, { ArrowDirection } from './TooltipArrow';
 interface IProps {
   data: any;
   tooltipOffset: TooltipOffset;
 }
 
+const useStyles = makeStyles({
+  root: {
+    '& .leaflet-popup-content-wrapper': {
+      borderRadius: 6,
```
Comment (yuvalbl) [inline src/components/atoms/AnywayMostSevereAccidentsMarker.tsx:17]: externalize to ``a global const named `defaultBorderRadius` in `styles/_globals.ts`. Use the value of 4.

File: src/components/atoms/AnywayMostSevereAccidentsMarker.tsx [hunk 2]
```diff
@@ -1,15 +1,52 @@
...
+      backgroundColor: '#e2e2e2f0',
```
Comment (yuvalbl) [inline src/components/atoms/AnywayMostSevereAccidentsMarker.tsx:18]: same as above - externalize to global as a new color in pallet. name variable as `lightBackgroundColor`

File: src/components/atoms/TooltipArrow.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+import { makeStyles } from '@material-ui/core';
```
Comment (yuvalbl) [inline src/components/atoms/TooltipArrow.tsx:2]: Nice refactoring :+1:

File: src/components/atoms/TooltipArrow.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+export enum ArrowDirection {
+  TOP,
```
Comment (yuvalbl) [inline src/components/atoms/TooltipArrow.tsx:5]: `TooltipOffset` and `ArrowDirection` enums have the same values.
You can use the same enum, and externalize it to util file `utils\enum.utils.ts`.
you can name the enum `direction` or [ClockPosition](https://en.wikipedia.org/wiki/Clock_position)

File: src/components/atoms/TooltipArrow.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    case ArrowDirection.BOTTOMLEFT: {
+      return 'rotate(-270deg)';
+    }
+  }
+};
+
+const useStyles = makeStyles({
+  arrow: {
+    border: 'solid black',
+    borderWidth: '0 2px 2px 0',
```
Comment (yuvalbl) [inline src/components/atoms/TooltipArrow.tsx:51]: Use width of `4px`

Comment (SejoB) [reply src/components/atoms/TooltipArrow.tsx]: Changed to 3px, 4 it's to much I think

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**